utopia-remix/package.json

```diff
@@ -5,9 +5,9 @@
   "type": "module",
   "scripts": {
     "build": "remix build",
-    "dev": "remix dev --manual",
+    "dev": "remix dev -c \"node express.mjs\"",
     "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
-    "start": "remix-serve ./build/index.js",
+    "start": "node build/server.js",
```

Comment:
I _think_ we should be adding the server file to the remix config: https://remix.run/docs/en/main/file-conventions/remix-config#server
I'm not sure if that magically saves us from having to manually provide the command to the dev server above, but it would at least include it in the output.
Speaking of, I think `express.mjs` needs to be renamed to `server.js` (or the `start` script here needs use `express.mjs`)